uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/device/settings/GoogleSettings.java

```diff
@@ -328,8 +333,14 @@ private void handleDoneButton() throws UiObjectNotFoundException {
         if (android.os.Build.VERSION.SDK_INT == 28) {
             UiAutomatorUtils.handleButtonClick("com.android.settings:id/redaction_done_button");
         } else {
-            final UiObject doneButton = UiAutomatorUtils.obtainUiObjectWithExactText("Done");
-            doneButton.click();
+            try {
+                final UiObject doneButton = UiAutomatorUtils.obtainUiObjectWithExactText("Done");
+                doneButton.click();
+            } catch (UiObjectNotFoundException e) {
+                Logger.i(TAG, "First Done button attempt failed: " + e.getMessage());
+                final UiObject doneButton = UiAutomatorUtils.obtainUiObjectWithExactText("DONE");
+                doneButton.click();
+            }
```

Comment:
Change this to conditional

Reply:
It looks like this is not actually the case for all API 32 devices on firebase. Most of the time we are actually still just getting "Done". I'm reverting back to the old design with try/catch, with the "Done" check first and "DONE" as a back up. (Wish there was a way to do this without case-sensitivity in the initial check but it doesn't look like it)